doc/api/util.md

```diff
@@ -215,7 +215,7 @@ by `util.inspect`. This is useful for functions like `console.log` or
 set to an object containing one or more valid [`util.inspect()`][] options.
 Setting properties directly is also supported.
 
-Returns the current options as an object.
+Returns the current default options as an object.
```

Comment:
Since this is a property and not a function, it doesn't really "return" anything, right? It just *is* a value. I'd be inclined to delete the line entirely. I don't think it actually adds any information. (I also think it's misleading. If you don't set it, it's `undefined` and not an object.)